mriqc/interfaces/functional.py

```diff
@@ -256,7 +256,7 @@ def _run_interface(self, runtime):
         if not isdefined(self.inputs.in_mask):
             _, mask_data, _ = auto_mask(func_data, nskip=self.inputs.skip_frames)
         else:
-            mask_data = nb.load(self.inputs.in_mask).get_data()
+            mask_data = nb.load(self.inputs.in_mask).get_fdata()
```

Comment:
I would assume masks are binary...

```suggestion
            mask_data = np.bool_(nb.load(self.inputs.in_mask).dataobj)
```

Or, you might choose `uint8` or `int16` or something else.